cli/Dockerfile

```
@@ -1,4 +1,5 @@
-FROM ghcr.io/scalar-labs/jre8:1.1.14
+FROM eclipse-temurin:8-jre
+RUN apt-get upgrade -y
```

```suggestion
RUN apt-get update && apt-get upgrade -y \
 && rm -rf /var/lib/apt/lists/*
```

I think we need to run the `apt-update` command before we install/upgrade packages.

Also, according to [the official document "Best practices for Dockerfile instructions"](https://docs.docker.com/develop/develop-images/instructions/#apt-get), I think it would be better to remove `apt cache` to reduce the container image size.

---

Thank you for the reminder.
It doesn't make any sense to do `apt-get upgrade` without doing `apt-get update` since the cache is out-to-dated.